src/views/RegisterPage/RegisterPage.js

```diff
@@ -0,0 +1,173 @@
+          divided="vertically"
+          columns={2}
+        >
+
+          <GridItem>
+            <Header
+              as="h2"
+              textAlign="center"
+            >
+                Sign In with Email.
+
+
+            </Header>
+            <Header
+              as="h4"
+              textAlign="center"
+            >
+                Enter the email address associated with your account,
+                and we’ll send a magic link to your inbox.
+            </Header>
+            {successMessage
+              ? (
+                <Message
+                  success
+                  header="Your user registration was successful"
+                  content={successMessage}
+                />
+              ) : ''}
+
+            <CustomForm
+              className="register-form"
+              loading={loading}
+              size="large"
+              errorMessage={errorMessage}
+              handleSubmit={this.handleSubmit}
+              buttonName="register"
+              inputFields={[
+                {
+                  fluid: true,
+                  name: 'username',
+                  id: 'username',
+                  placeholder: 'username',
+                  type: 'text',
+                  value: username,
+                  onChange: this.handleInputChange,
+                  error: !isEmpty(errorMessage ? errorMessage.username : ''),
+                },
+                {
+                  fluid: true,
+                  name: 'email',
+                  placeholder: 'E-mail address',
+                  type: 'email',
+                  value: email,
+                  onChange: this.handleInputChange,
+                  error: !isEmpty(errorMessage ? errorMessage.email : ''),
+                },
+                {
+                  fluid: true,
+                  name: 'password',
+                  placeholder: 'Password',
+                  type: 'password',
+                  value: password,
+                  onChange: this.handleInputChange,
+                  error: !isEmpty(errorMessage ? errorMessage.password : ''),
+                },
+              ]}
+            />
+            <Message warning>
+              <Icon name="help"/>
+                Already signed up?&nbsp;
+                login
+                &nbsp;instead.
+            </Message>
+          </GridItem>
+        </GridContainer>
+      </div>
+    );
+  }
+}
+
+RegisterPage.defautProps = {
+  loading: false,
+};
+
+RegisterPage.propTypes = {
+  // eslint-disable-next-line max-len
+  // eslint-disable-next-line
+  // react/forbid-prop-types,react/require-default-props,react/no-unused-prop-types
+  errorMessage: PropTypes.object,
+  // eslint-disable-next-line react/require-default-props
+  successMessage: PropTypes.string,
+  // eslint-disable-next-line react/require-default-props
+  loading: PropTypes.bool,
+  signUpAction: PropTypes.func.isRequired,
+};
+
+export const mapStateToProps = ({ auth }) => ({
```

Comment:
You don't have to export this function if it's only being used within the file.

Reply:
I exported the function to test if it is working as expected.